packages/ui/src/memberships/components/SocialMediaSelector.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback, useState } from 'react'
+import { useFormContext } from 'react-hook-form'
+import styled from 'styled-components'
+
+import { InputComponent, InputText } from '@/common/components/forms'
+import { TextBig, TextMedium } from '@/common/components/typography'
+import { capitalizeFirstLetter } from '@/common/helpers'
+import { socialMediaList, SocialMediaTile, Socials } from '@/memberships/components/SocialMediaTile/SocialMediaTile'
+import { MemberFormFields } from '@/memberships/modals/BuyMembershipModal/BuyMembershipFormModal'
+
+const socialToPlaceholder: Record<Socials, string> = {
+  HYPERLINK: 'Enter URL',
+  WECHAT: 'Enter Username',
+  IRC: 'Enter Username',
+  WHATSAPP: 'Enter Username',
+  MATRIX: 'Enter Username',
+  YOUTUBE: 'Enter channel name',
+  FACEBOOK: 'Enter Username',
+  DISCORD: 'Enter Member handle',
+  TELEGRAM: 'Enter Username',
+  TWITTER: 'Enter Username',
+  EMAIL: 'Enter Email',
+}
+
+interface Props {
+  initialSocials?: Socials[]
+}
+
+export const SocialMediaSelector = ({ initialSocials }: Props) => {
```

Comment:
Wasn't this file part of the other PR ?

Reply:
Yeah, it was. Idk why it didn't get removed on merge,